packages/beacon-node/src/execution/builder/http.ts

```diff
@@ -37,6 +37,12 @@ export const defaultExecutionBuilderHttpOpts: ExecutionBuilderHttpOpts = {
   timeout: 12000,
 };
 
+/**
+ * Duration given to the builder to provide a `SignedBuilderBid` before the deadline
+ * is reached, aborting the external builder flow in favor of the local build process.
+ */
+const BUILDER_PROPOSAL_DELAY_TOLERANCE = 1000;
```

Comment:
this is too tight, 1.5 sec seems more reasonable

Reply:
mev-boost uses a default value of 950ms (slightly lower to account for passing response to CL), this is what all other CLs use and the spec defines

Reply:
as discussed on [discord](https://discord.com/channels/593655374469660673/1288837426696945696/1294287747707699271), this is as per spec